### PR TITLE
Feature/3623 update UI test

### PR DESCRIPTION
**Pull request by puneetmahali1**
## Description
This PR is for the fixing new submission flow UI Test and the new Thank You screen UI Test.
## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3623

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift [hunk 1]
```diff
@@ -33,6 +33,10 @@ class ExposureSubmissionTestResultViewController: DynamicTableViewController, EN
 
 		setUpView()
 		setUpBindings()
+		
+		footerView?.primaryButton?.accessibilityIdentifier = AccessibilityIdentifiers.ExposureSubmission.primaryButton
+		footerView?.secondaryButton?.accessibilityIdentifier = AccessibilityIdentifiers.ExposureSubmission.secondaryButton
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift:38]: I wonder if we cannot simply set a default accessibility ID for the footer buttons. They are unique per screen.
@nickguendling @marcussc ?

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift [hunk 2]
```diff
@@ -280,7 +313,54 @@ class ENAUITests_04_ExposureSubmissionUITests: XCTestCase {
 		XCTAssertTrue(app.buttons["AppStrings.ExposureSubmission.secondaryButton"].waitForExistence(timeout: .medium))
 		app.buttons["AppStrings.ExposureSubmission.secondaryButton"].tap()
 
-		// NewSubmissionFlow: Add the test for the cancel Alert
+		app.alerts["Sie wollen keine Warnung verschicken?"].buttons["Nicht warnen"].tap()
+		
+		waitForNonExistence(of: app.buttons["AppStrings.ExposureSubmission.primaryButton"], timeout: .long)
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift:318]: I read it (and the extension) but I don't get it. What do you want to test here?

Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift:318]: On Test Result Screen there is alert popup. So, I am checking the flow with the popups buttons.

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift (hunk 1 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift:38]: Sounds good 👍

Comment (whiskey) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift:38]: Will be added later.

**Review by whiskey — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift (hunk 1 above)
Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift:38]: Yes

**Review by marcussc — APPROVED** [review]

**Pull request merged**